fastify.js

```diff
@@ -60,16 +60,16 @@ const onBadUrlContext = {
 }
 
 function defaultErrorHandler (error, request, reply) {
-  if (reply.statusCode >= 500) {
-    reply.log.error(
-      { req: request, res: reply, err: error },
-      error && error.message
-    )
-  } else if (reply.statusCode >= 400) {
+  if (reply.statusCode < 500) {
```

Comment:
The title of this PR is about improving test coverage. This change does not seem to be related to that goal. Please refrain from doing multiple unrelated things in a single PR.

Reply:
It actually is. With the `if` and `else if`, the coverage report expects a case that does not lie in either of the blocks. However, it's not possible here because of [this line](https://github.com/fastify/fastify/blob/master/lib/reply.js#L504). 

So changing it to if/else indeed increases coverage.

Reply:
I think this is actually different behavior. Is it possible to get an error object in this function with a status code that is not `4xx` or `5xx`.

Reply:
That's what I tried, but that line I linked above converts all < 400 codes to 500. 
Do you have a way to work around that?